crates/polars-core/src/chunked_array/temporal/time.rs

```diff
@@ -23,6 +23,7 @@ impl TimeChunked {
     pub fn to_string(&self, format: &str) -> StringChunked {
         let mut ca: StringChunked = self.apply_kernel_cast(&|arr| {
             let mut buf = String::new();
+            let format = if format == "iso" { "%T%.9f" } else { format };
```

Comment:
@alexander-beedie This format `%T%.9f` is not an iso format.
Is it appropriate to use `iso` as an alias for something that is not an ISO format?
It seems that the true ISO format can be specified with `%+`.
https://docs.rs/chrono/latest/chrono/format/strftime/index.html#fn7